### ISSUE TITLE
Adding licensing information to the code

### ISSUE THREAD

**Issue by abn** [victims/victims-lib-java#8]
@dfj mentioned AGPL v3 for our use - http://www.gnu.org/licenses/agpl-3.0.txt

@ashcrow @gcmurphy @kholia got input on this?

**Comment by ashcrow** [comment]
:+1:

**Comment by gcmurphy** [comment]
:white_check_mark:

**Comment by abn** [comment]
Completed adding AGPL v3.

**Comment by abn** [comment]
This is the generated header file. Let me know if you have any thoughts/concerns. @ashcrow @dfj @gcmurphy 

I also want to confirm the use of "The Victims Project" is okay or should we be using "Red Hat Inc."

``` java
/*
 * #%L
 * This file is part of victims-lib.
 * %%
 * Copyright (C) 2013 The Victims Project
 * %%
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 * 
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a copy of the GNU Affero General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 * #L%
 */
```

**Comment by ashcrow** [comment]
That header looks good to me. :+1:

**Comment by abn** [comment]
Released 1.0. #1
